src/Context/AuthContext.js

```diff
@@ -0,0 +1,60 @@
+import { createContext, useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  let navigate = useNavigate();
+
+  const tokenAvailable = localStorage.getItem("token") ? true : false;
+
+  const [isLoggedIn, setIsLoggedIn] = useState(tokenAvailable);
+
+  const loginHandler = async (email, password) => {
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email: email,
+        password: password,
+      });
+      localStorage.setItem("token", response.data.encodedToken);
+      setIsLoggedIn(true);
+      navigate(`/product-lists`);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const signupHandler = async ({
+    firstName,
+    lastName,
+    email,
+    password,
+    confPassword,
+  }) => {
+    try {
+      const response = await axios.post(`/api/auth/signup`, {
+        firstName: firstName,
+        lastName: lastName,
+        email: email,
+        password: password,
+        confPassword: confPassword,
```

Comment:
Since you are using the same variable names for the object, you can only write the parameters and pass it normally. 
Like : 
 `{
        firstName,
        lastName,
        email,
        password, 
        confPassword
 }`

Reply:
Thank you for pointing this out. I'll make changes